src/util/handlebars/handlebar-helpers.ts

```diff
@@ -59,4 +60,9 @@ export const registerHandlebarsHelpers = () => {
 			subscriptionHost !== jiraHost
 	);
 
+	hbs.registerHelper("toDay", (date: Date | null | undefined) => {
+		if (!date) return "";
+		return format(date, "YYYY-MM-DD");
+	});
```

Comment:
Is this correct date format we're using at other places? I think we're using `mm-dd-yyy`

Reply:
Tests added. Yup I think 'YYYY-MM-DD' is the correct format https://date-fns.org/v1.29.0/docs/format

Reply:
This date format is different from we're using in restart backfill window.

<img width="577" alt="Screenshot 2023-03-01 at 9 22 37 am" src="https://user-images.githubusercontent.com/14277763/221995041-0f48be90-2ec9-4164-b35d-d9ebdaa3108b.png">

Reply:
Oh, did you mean UX problem instead of bug?
UX: Yup it is confusing. However I think the ISO format is better than some au format mm-dd-yyy, it won't make sense to us folks.
Bug wise: I checked the date sent from js to backend is ISO format YYYY-MM-DD.

Reply:
This is how we initialise the current date to restart backfill date input:

```
const initializeBackfillDateInput = function () {
	const dateElement = document.getElementById('backfill-date-picker');
	const date = new Date();
	date.setFullYear(date.getFullYear() - 1);
	dateElement.value = date.toISOString().split('T')[0];
	dateElement.max = new Date().toISOString().split('T')[0];
}
```

Reply:
Hey @Harminder84 I've created a followup ticket for the localised ui label. https://softwareteams.atlassian.net/browse/ARC-2008

So closing this now.